CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## [1.0.1] - 2021-08-02
+
+### Added
+- A new `tpm_devid` NodeAttestor plugin is now available (#2111, #2427)
```

Comment:
Although it can be inferred from the plugin name, I think that it would be good to have something in this line that briefly describes what kind of node attestation is being introduced. Probably indicating that's a plugin that attests nodes that own a TPM and that have been provisioned with a DevID certificate or something along those lines.

Reply:
How about this

```suggestion
- LDevID-based TPM attestation can now be performed via a new `tpm_devid` NodeAttestor plugin (#2111, #2427)
```